crates/extensions/c8y_mapper_ext/src/config_operations.rs

```diff
@@ -151,44 +155,37 @@ impl CumulocityConverter {
                 let event_response_id = self.http_proxy.send_event(create_event).await?;
 
                 // Send a request to the Uploader to upload the file asynchronously.
-                let uploaded_file_path = self
-                    .config
-                    .data_dir
-                    .file_transfer_dir()
-                    .join(target.external_id.as_ref())
-                    .join("config_snapshot")
-                    .join(format!(
-                        "{}-{}",
-                        response.config_type.replace('/', ":"),
-                        cmd_id
-                    ));
-
-                let binary_upload_event_url = self
-                    .c8y_endpoint
-                    .get_url_for_event_binary_upload_unchecked(&event_response_id);
-
-                let upload_request = UploadRequest::new(
-                    self.auth_proxy
-                        .proxy_url(binary_upload_event_url.clone())
-                        .as_str(),
-                    &uploaded_file_path,
-                );
+                let config_filename =
+                    format!("{}-{}", response.config_type.replace('/', ":"), cmd_id);
 
-                self.uploader_sender
-                    .send((cmd_id.into(), upload_request))
-                    .await
-                    .map_err(CumulocityMapperError::ChannelError)?;
+                let uploaded_file_url = format!(
```

Comment:
Minor. At my first look, I got confused because of the mix usage of "upload" and "downlonad". I think changing the variable name to `tedge_file_url` is better to understand as the url is localhost url.

```suggestion
                let tedge_file_url = format!(
```

Reply:
would `fts_file_url` work since we're donwloading from the FTS?

Reply:
> would `fts_file_url` work since we're donwloading from the FTS?

Yes, but actually this url is exactly the same as `tedgeUrl` in a command payload.

Reply:
> Yes, but actually this url is exactly the same as `tedgeUrl` in a command payload.

Ah, I see, it makes sense to call it `tedge_file_url` then.